### PR TITLE
Expose Environment.GetCommandLineArgs

### PR DESCRIPTION
**Pull request by pallavit**
Expose Environment.GetCommandLineArgs in the System.Runtime.Extensions contract and add tests for the same.

### CODE REVIEW

**Comment by pallavit** [comment]
@stephentoub, @weshaggard Could you please take a look at this.

**Comment by pallavit** [comment]
#2413

File: src/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            Assert.True(cmdLineArgs[3].Contains("RemoteInvoke"));
+
+            // Check the arguments sent to the method.
+            Assert.True(cmdLineArgs.Length - 4 == args.Length);
+            for (int i = 0; i < args.Length; i++)
+            {
+                Assert.Equal(args[i], cmdLineArgs[i + 4]);
+            }
+
+            return 42;
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs]:
```C#
return SuccessExitCode;
```

File: src/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            }
+
+        }
+
+        public static int CheckCommandLineArgs(string[] args)
+        {
+
+            string[] cmdLineArgs = Environment.GetCommandLineArgs();
+
+            Assert.True(cmdLineArgs.Length >= 4); /*AppName, AssemblyName, TypeName, MethodName*/
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs]:
```C#
Assert.InRange(cmdLineArgs.Length, 4, int.MaxValue);
```
That way we'll get the actual value in the assert message if it fails.

File: src/System.Runtime.Extensions/tests/project.lock.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/project.lock.json]: Was this supposed to change?

**Comment by stephentoub** [comment]
A few minor comments, but otherwise LGTM.

File: src/System.Runtime.Extensions/tests/project.lock.json (hunk 3 above)
Comment (pallavit) [inline src/System.Runtime.Extensions/tests/project.lock.json]: Huh, this happened automatically. Bug? I will force update this to true.

**Comment by stephentoub** [comment]
LGTM

**Comment by pallavit** [comment]
Thanks for the review.

**Pull request merged**